custom/enikshay/tests/utils.py

```diff
@@ -98,73 +99,84 @@ def occurrence(self):
 
     @property
     def episode(self):
```

Comment:
FYI @proteusvacuum I changed the the way `ENikshayCaseStructureMixin` works slightly so that tests can modify the episode before calling `create_case_structure()